org.librarysimplified.audiobook.tests.device/src/androidTest/java/org/librarysimplified/audiobook/tests/device/MockPlayerActivity.kt

```diff
@@ -62,7 +62,8 @@ class MockPlayerActivity : AppCompatActivity(), PlayerFragmentListenerType {
   override fun onCreate(state: Bundle?) {
     super.onCreate(state)
 
-    this.setTheme(R.style.AudioBooksWithActionBar)
+    // this.setTheme(R.style.AudioBooksWithActionBar)
+    this.setTheme(R.style.SimplifiedTheme_ActionBar)
```

Comment:
What is going on here? Is the current actionbar not styled correctly?

Reply:
Ah yeah this was to fix the PlayerFragmentTest which was failing due to having a style attribute specified in the layout that wasn't defined in the styles created in this test module. I figured rather than adding to the test-specific style we should just use the same style we use to build the app. 

I'll remove the commented line though.